addons/static_radios/functions/fnc_moduleStaticRadio.sqf

```diff
@@ -29,7 +29,7 @@ private _unit = (attachedTo _logic);
 private _classname =  ((getItemCargo _unit) select 0) select 0;
 
 if !((_unit call TFAR_fnc_isLRRadio) || {(_classname call TFAR_fnc_isPrototypeRadio)} || {(_classname call TFAR_fnc_isRadio)}) exitWith {
-    hint "This has to be used on a Radio";
+    hint "$STR_TFAR_Zeus_moduleStaticRadio_hint";
```

Comment:
This doesn't work.. I didn't even think of it in the previous ones. Inside scripts you have to use `localize` command

Reply:
Ok.